inc/TRestRawSignalShapingProcess.h

```diff
@@ -47,10 +45,12 @@ class TRestRawSignalShapingProcess : public TRestEventProcess {
     // add here the members of your event process
     TString fResponseFilename;
 
-    TString fShapingType;
-
-    Double_t fShapingTime;
-    Double_t fShapingGain;
+    /// Types are : gaus, shaper, shaperSin, responseFile
+    TString fShapingType = "shaperSin";
```

Comment:
Maybe we can put the initial values in `TRestRawSignalShapingProcess::Initialize()`.

Reply:
I thought writting the default values at the header was a good option. What would be the best? Having them always at initialize? 

In that case, I believe if would be possible to have a simple python script that checks that inside the `Initialize` method we find all the members defined at the header. At least the ones that have `//<`.

Reply:
I don't know what is better but from the codes I read I seldom find the initial values defined in the header.

Reply:
Right, I am migrating it now. Because when we remove the `InitFromConfigFile` method we lose the default values given to the method, and therefore I was assigning it at the header.

The `Initialize` method is private and just being called at the constructor, so I believe that initializing the members at the header will just simplify the initialize method to input/output event initialization and library and classname definitions.